### PR TITLE
Fixes a security issue

### PR DESCRIPTION
**Pull request by C0kkie**
#### Changes proposed in this pull request: 
- In general it is theoretically possible to execute every script on the server via a hackmenu or some other shit, because bis_fnc_execvm was approved for some reasons in the cfgremotexec, which is not needed.

- [ ] I have tested my changes and it worked as expected, the only thing what i couldn't test is with headless client

### CODE REVIEW

File: Altis_Life.Altis/initPlayerServer.sqf [hunk 1]
```diff
@@ -1,10 +0,0 @@
-#include "script_macros.hpp"
-/*
-    File: initPlayerServer.sqf
-    Author:
-
-    Description:
-    Executed only on server when a player joins mission.
-*/
-if (LIFE_SETTINGS(getNumber,"player_deathLog") isEqualTo 0) exitWith {};
```
Comment (DomT602) [inline Altis_Life.Altis/initPlayerServer.sqf]: We've lost whether to enable/disable it with your new fix.

Comment (C0kkie) [reply Altis_Life.Altis/initPlayerServer.sqf]: Done

**Comment by DomT602** [comment]
@C0kkie sorry for the lack of activity, as pointed out in the discord 'BIS_fnc_execVM' also needs to be removed from SpyGlass. Thanks for the previous fix though!

**Comment by C0kkie** [comment]
Should be done @DardoTheMaster

**Comment by Turdant** [comment]
You forgot `bis_fnc_execvm_meta` on line 163 in _Config_SpyGlass.hpp_

**Comment by C0kkie** [comment]
done

**Pull request merged**